src/routes/landing/components/CitySelector.js

```diff
@@ -42,7 +43,7 @@ class CitySelector extends React.PureComponent<PropsType> {
         .filter(_city => {
           const isCityName = _city.name.toLowerCase().includes(filterText)
           const isAlias = _city._aliases && Object
-            .keys(_city._aliases).some(alias => alias.toLowerCase().includes(filterText))
+            .keys(_city._aliases).some(alias => normalize(alias).toLowerCase().includes(filterText))
```

Comment:
`toLowerCase` missing

Reply:
Maybe this would be a good time to refactor this into a separate function and reuse it.

Reply:
let me create a separate ticket for it